source/includes/webhooks/_introduction.md

```diff
@@ -205,13 +205,13 @@ sidekiq_retry_in do |index, _exception|
 end
 ```
 
-The table below outlines the estimated wait time for each retry request, assuming that rand(30) always returns 0.
+The table below outlines the estimated wait time for each attempt, assuming that rand(30) always returns 0.
 
-| Retry number | Next Retry in | Total waiting time |
-| ------------ | ------------- | ------------------ |
-|       1      |        1m     |        0h  1m      |
-|       2      |       15m     |        0h 16m      |
-|       3      |       60m     |        1h 16m      |
-|       4      |      120m     |        3h 16m      |
-|       5      |      240m     |        7h 16m      |
-|       6      |      480m     |       15h 16m      |
+| Attempt number | Next Attempt in | Total waiting time  |
+| -------------- | --------------- | ------------------  |
+|       1        |        1m       |            0m       |
+|       2        |       15m       |        0h  1m       |
+|       3        |       60m       |        0h 16m       |
+|       4        |      120m       |        1h 16m       |
+|       5        |      240m       |        3h 16m       |
+|       6        |        --       |        7h 16m       |
```

Comment:
Would it make more sense to put the last column first?
```suggestion
| Approx. Total waiting time | Attempt number | Next Attempt in | 
| -------------- | --------------- | ------------------  |
|   0m         |   1        |        1m       | 
|   0h  1m   |   2        |       15m       |       
|   0h 16m  |   3        |       60m       |        
|   1h 16m   |   4        |      120m       |        
|   3h 16m  |   5        |      240m       | 
|   7h 16m  |   6        |        --       |  
```

Reply:
Thanks, this is done also!